by setting the policy right at the beginning the booting via NFS isn't possible
the policy will set later on anyway (at a point of time where the local access is allowed)

diff --git a/package/firewall/patches/000-allow-boot-via-nfs.patch b/package/firewall/patches/000-allow-boot-via-nfs.patch
new file mode 100644
--- /dev/null
+++ b/package/firewall/patches/000-allow-boot-via-nfs.patch
@@ -0,0 +1,11 @@
+--- a/defaults.c
++++ b/defaults.c
+@@ -397,7 +397,7 @@ fw3_flush_rules(struct fw3_ipt_handle *h
+ 	if (!has(defs->flags, handle->family, handle->table))
+ 		return;
+ 
+-	if (handle->table == FW3_TABLE_FILTER)
++	if (handle->table == FW3_TABLE_FILTER && policy == FW3_FLAG_ACCEPT)
+ 	{
+ 		fw3_ipt_set_policy(handle, "INPUT",   policy);
+ 		fw3_ipt_set_policy(handle, "OUTPUT",  policy);
